portal/portal-util/util/src/java/org/sakaiproject/portal/util/CSSUtils.java

```diff
@@ -126,10 +158,32 @@ public static String getCssToolSkin(Site site)
 	 */
 	public static String getCssToolSkinCDN(Site site)
 	{
-		String skinFolder = site.getSkin();
+		String skinFolder=null;
+		if (site!=null) {
+			skinFolder = site.getSkin();
+		}
 		return getCssToolSkinCDN(skinFolder);
 	}
 
+	public static String getCssHead(String skin, boolean isInlineRequest) {
+		// setup html information that the tool might need (skin, body on load,
+		// js includes, etc).
+		String headCssPortalSkin = "<link href=\"" 
+				+ getCssPortalSkinCDN(skin)
+				+ "\" type=\"text/css\" rel=\"stylesheet\" media=\"all\" />\n";
```

Comment:
May you change this two `media="all"` for `media="screen, tty, tv, handheld, projection"`as I did for morpheus velocity? #1843 This would be to avoid misplaces on printing Sakai.